servers/register_server_types.cpp

```diff
@@ -100,18 +101,18 @@ void register_server_types() {
 	ClassDB::register_virtual_class<DisplayServer>();
 	ClassDB::register_virtual_class<RenderingServer>();
 	ClassDB::register_class<AudioServer>();
-	ClassDB::register_virtual_class<PhysicsServer3D>();
 	ClassDB::register_virtual_class<PhysicsServer2D>();
+	ClassDB::register_virtual_class<PhysicsServer3D>();
+	ClassDB::register_virtual_class<NavigationServer2D>();
+	ClassDB::register_virtual_class<NavigationServer3D>();
 	ClassDB::register_class<XRServer>();
 	ClassDB::register_class<CameraServer>();
 
+	ClassDB::register_virtual_class<RenderingDevice>();
+
 	ClassDB::register_virtual_class<XRInterface>();
 	ClassDB::register_class<XRPositionalTracker>();
 
-	ClassDB::add_compatibility_class("ARVRServer", "XRServer");
-	ClassDB::add_compatibility_class("ARVRInterface", "XRInterface");
-	ClassDB::add_compatibility_class("ARVRPositionalTracker", "XRPositionalTracker");
```

Comment:
Moved to `scene/register_scene_types.cpp` for convenience as I'm consolidated everything there, even `server/` classes.